src/viewer.js

```diff
@@ -2530,6 +2530,7 @@ function onCanvasDblClick( event ) {
     });
 }
 
+// Modified to improve constrained panning
```

Comment:
Thanks for the info. I think this should be removed from the code before being merged though.